### PR TITLE
Add bank handler for `VIRGIN_NRNBGB22` (Virgin Money)

### PR DESCRIPTION
**Pull request by matt-fidd**
This change adds handling in for Virgin Money (VIRGIN_NRNBGB22). They don't return `creditorName` or `debtorName` through GoCardless so all transactions use the unprocessed `remittanceInformationUnstructured` field which contains extra details that make creating rules harder.

### CODE REVIEW

**Review by psybers — CHANGES_REQUESTED** [review]
Can you add a test case? See https://github.com/actualbudget/actual-server/blob/master/src/app-gocardless/banks/tests/belfius_gkccbebb.spec.js for an example.

File: src/app-gocardless/banks/virgin_nrnbgb22.js
Comment (psybers) [inline src/app-gocardless/banks/virgin_nrnbgb22.js:15]: Should this be anchored to the start of the string?

Should it be `\d+` instead of 0 or more digits?

**Review by psybers — CHANGES_REQUESTED** [review]

File: src/app-gocardless/banks/virgin_nrnbgb22.js
Comment (matt-fidd) [reply src/app-gocardless/banks/virgin_nrnbgb22.js]: You're right, fixed

**Comment by psybers** [comment]
Be sure to update this file too: https://github.com/actualbudget/actual-server/blob/master/src/app-gocardless/tests/bank-factory.spec.js

**Comment by psybers** [comment]
Actually hold off on modifying the test for bank factory. #367 should make that no longer necessary.

**Comment by MatissJanis** [comment]
Thanks for working on this! Would you mind updating the PR to use prop spreading for `Fallback`? Here's an example bank provider: https://github.com/actualbudget/actual-server/blob/aa22e6951dd90fc03e086593f001d0a002b6e324/src/app-gocardless/banks/american-express-aesudef1.js#L7

We recently merged some changes to simplify them.

Happy to merge this PR once that's done + the conflict is solved!

**Comment by matt-fidd** [comment]
> Thanks for working on this! Would you mind updating the PR to use prop spreading for `Fallback`? Here's an example bank provider:
> 
> https://github.com/actualbudget/actual-server/blob/aa22e6951dd90fc03e086593f001d0a002b6e324/src/app-gocardless/banks/american-express-aesudef1.js#L7
> 
> We recently merged some changes to simplify them.
> 
> Happy to merge this PR once that's done + the conflict is solved!

All done! Thank you

**Review by MatissJanis — APPROVED** [review]
Thanks!

**Pull request merged**
